### DISCUSSION TITLE
Can I use Unpoly in Django?

### DISCUSSION THREAD

**Discussion by aliahadmd** [unpoly/unpoly#626] [General]
Hi, I am very new to Unpoly. I want to use it in Django. But it seems the Django Unpoly library is not maintainable. Can I still use it? Or can I use vanilla Unpoly for my Django website? 
Thanks.

**Comment by triskweline** [comment]
Hi @aliahadmd. You don't need a Django library to use Unpoly. Simply load `unpoly.js` and `unpoly.css` via [npm](https://unpoly.com/install/npm) or a [CDN](https://unpoly.com/install/cdn).

**Reply by aliahadmd** [reply]
Thanks for your response! That sounds pretty straightforward. I'll give it a try .

**Comment by davisums** [comment]
In case you're talking about the server protocol implementation. I'm using this library and it's up to date and working as expected: https://gitlab.com/rocketduck/python-unpoly

**Reply by aliahadmd** [reply]
Thanks, I will try.

**Comment by Macktireh** [comment]
Yes, absolutely! You don't need a Django library to use Unpoly. You just need to include the Unpoly JavaScript and CSS files, and you're good to go.

If you want to see how I explored Django and Unpoly in a CRUD project with dynamic page navigation recently, here's the link: [django-unpoly-project](https://github.com/Macktireh/django-unpoly-project)

**Reply by aliahadmd** [reply]
Thanks for sharing. I'll definitely check it out.

**Comment by alexandremjacques** [comment]
I have a blog post on how to install Unpoly in a Django project: https://alexandremjacques.com/django-plus-unpoly-part-1/
